charts/datadog/README.md

```diff
@@ -153,7 +153,7 @@ See [0.18.1's README](https://github.com/helm/charts/blob/847f737479bb78d89f8fb6
 To uninstall/delete the `<RELEASE_NAME>` deployment:
 
 ```bash
-helm delete <RELEASE_NAME> --purge
+helm uninstall <RELEASE_NAME> 
```

Comment:
the file https://github.com/DataDog/helm-charts/blob/main/charts/datadog/README.md.gotmpl should be update and then the `README.md` will be updated when running `.github/helm-docs.sh`